sh/setup-migration-script.sh

```diff
@@ -14,8 +14,12 @@ set -uo pipefail
 
 echo "[migrate] Migrating BHIMA database"
 
+ENVIRONMENT=${ENVIRONMENT:-"production"}
+
+echo "[migrate] Reading from $ENVIRONMENT."
+
 set -a
-source .env.development
+source .env.$ENVIRONMENT
```

Comment:
I would like to know where to set the `ENVIRONMENT` if we want to run in production or in development mode

Reply:
On line 17, it is set to "production" by default.  However you can change this the same way you set any environmental variable in bash, by just passing it in.  Like this:
```sh
ENVIRONMENT="development" yarn migrate
# now ENVIRONMENT will be "development"
```
or 
```sh
exports ENVIRONMENT="development"
yarn migrate
```
etc.